File: BE/health-service/src/main/java/com/s206/health/nutrition/statistics/dto/response/NutritionStatsEntry.java
package com.s206.health.nutrition.statistics.dto.response;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class NutritionStatsEntry {
    private String label;
    private Integer calorie;
    private Integer carbs;
    private Integer protein;
    private Integer fat;
    private Integer sugar;

    public Integer getTotal() {
        return carbs + protein + fat + sugar;
    }
}

File: FE/mobile/src/main/java/com/example/diaviseo/network/meal/dto/res/NutritionStatsEntry.kt
package com.example.diaviseo.network.meal.dto.res

data class NutritionStatsEntry(
    val label: String,
    val calorie: Int,
    val carbs: Int,
    val protein: Int,
    val fat: Int,
    val sugar: Int,
    val total: Int?
)